modules/docs/mdx/7.0-MIGRATION-GUIDE.mdx

```diff
@@ -36,244 +39,320 @@ from Canvas Kit v6 to v7:
 > Note: This codemod only works on `.js`, `.jsx`, `.ts`, and `.tsx` extensions. You may need to make
 > some manual changes in other file types (`.json`, `.mdx`, `.md`, etc.).
 
-> Note: You may need to run your linter after executing the codemod, as it's resulting formatting
+> Note: You may need to run your linter after executing the codemod, as its resulting formatting
 > (spacing, quotes, etc.) may not match your project's styling.
 
-**Breaking changes accounted for by this codemod will be marked with a 🤖.**
+**Breaking changes handled by this codemod will be marked with a 🤖.**
 
 **Please verify all changes made by the codemod. As a safety precaution, we recommend committing the
 changes from the codemod as a single isolated commit (separate from other changes) so you can
 rollback more easily if necessary.**
 
 [Let us know](https://github.com/Workday/canvas-kit/issues/new?labels=bug&template=bug.md) if you
-encounter any issues or use cases that we've missed. The `@workday/canvas-kit-codemod` package will
-help us maintain additional codemod transforms to make future migrations easier.
+encounter any issues or use cases that we've missed.
 
-## Component Deprecations
+## General Changes
 
-### Deprecation Types
+### Dependency Upgrades
+
+We've upgraded to Emotion 11 and React 17 in v7. This allows teams to upgrade to newer versions of
+Emotion and React without needing to maintain older versions in order to use Canvas Kit.
```

Comment:
I lifted the Dependency Upgrades section from the Canvas Site (What's New > Announcements, see `Emotion 11, React 17, and Typescript 4.1 Upgrade`). I made a few modifications, but it's mostly the same.